File: src/main/java/com/gmail/borlandlp/minigamesdtools/activepoints/reaction/ReactionCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.activepoints.reaction;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class ReactionCreatorHub extends CreatorHub {
    public Reaction createReaction(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (Reaction) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/activepoints/reaction/ReactionCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.activepoints.reaction

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class ReactionCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): Reaction {
        return super.create(itemID, dataProvider) as Reaction
    }
}